### DISCUSSION TITLE
Publish public website despite private repository on Github

### DISCUSSION THREAD

**Discussion by Santonia27** [quarto-dev/quarto-cli#7985] [Q&A] [answered]
### Description

Is it possible to keep the Github Repository in which the docs are located private but only publish the docs website to the public?

**Comment by mcanouil** [comment, marked as answer]
This is independent of Quarto.
You should check GitHub documentation for the different account type (free, pro, etc.).

See <https://docs.github.com/en/pages>.


PS: I can already tell you that you need a "pro" subscription at least.
